src/PointPatterns.jl

```diff
@@ -25,6 +25,10 @@ export
   # thinning methods
   AbstractThinning,
   RandomThinning,
-  thin
+  thin,
+
+  # sampling methods
+  DiscretizedSampling,
+  ThinnedSampling
```

Comment:
```suggestion
  thin
```
I think these sampling algorithms are internal implementation details right? Not intended for end-users.